cmd/tsdb/main.go

```diff
@@ -383,3 +409,129 @@ func getFormatedTime(timestamp int64, humanReadable *bool) string {
 	}
 	return strconv.FormatInt(timestamp, 10)
 }
+
+func analyzeBlock(b *tsdb.Block, limit int) {
```

Comment:
Can we have single line comment at the beginning of each subsection of the code about what it is trying to calculate (in addition to `Printf` at the ends)? (makes the code easier to follow)

Reply:
I'm not sure that'd help much tbh